egov.gemspec

```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'webmock'
 
   # spec.add_runtime_dependency 'signer'
-  spec.add_runtime_dependency 'activesupport'
+  spec.add_runtime_dependency 'activesupport', '~> 5.2'
```

Comment:
Would you please relax activesupport version to ">= 5.2", since we use this gem in Rails 6.

Reply:
:(

Reply:
fixed: https://github.com/kufu/kiji/pull/15/commits/943eaec4c5f20c8518f18daebbc4539c47acca08